package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-demo-payment-web",
   "description": "Digital service mock to view scheduled payments for financial aid in the event a property subsides into mine shaft.",
-  "version": "1.19.4",
+  "version": "1.19.5",
   "homepage": "https://github.com/DEFRA/ffc-demo-payment-web",
   "repository": {
     "type": "git",
```

Comment:
AppInsights here is version `1.8.2` we're using `2.3.3` in other repos -- let's update

Reply:
Be worth checking the other dependencies against our de-facto versions in other repos

Reply:
compared vs payment service repo and updated